Mathlib/Data/Set/Card.lean

```diff
@@ -149,6 +149,10 @@ theorem encard_le_coe_iff {k : ℕ} : s.encard ≤ k ↔ s.Finite ∧ ∃ (n₀
   ⟨fun h ↦ ⟨finite_of_encard_le_coe h, by rwa [ENat.le_coe_iff] at h⟩,
     fun ⟨_,⟨n₀,hs, hle⟩⟩ ↦ by rwa [hs, Nat.cast_le]⟩
 
+@[simp]
+theorem encard_prod : (s ×ˢ t).encard = s.encard * t.encard := by
```

Comment:
Please also add `ncard_prod` (should be `by simp [ncard]` after this lemma). Otherwise LGTM. Thanks for turning this into a PR!

maintainer delegate